src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 0.419260215499674.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1897
1897:             $pathologicalRatio | Should -BeGreaterThan 10
```